java/src/main/java/com/cisco/pxgrid/samples/ise/StompSubscription.java

```diff
@@ -34,6 +34,7 @@ public StompFrame getSubscribeMessage() {
 		message.setCommand(StompFrame.Command.SUBSCRIBE);
 		message.setHeader("destination", topic);
 		message.setHeader("id", id);
+//		message.setHeader("filter", "nasIpAddress=='10.0.0.2'");
```

Comment:
Java is incomplete?

Reply:
For Java, I'm hardcoding since we cannot pass filter as an extra parameter. In maven dexec.args works as shell syntax and many of the filter characters like single quote, space, and back quote will not work properly.